File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) 
    {
        int n = nums.size();
        int sum = 0;
        vector<int> res(n);
        for(int i = 0;i<n;i++)
        {
            int prod = 1;
            for(int j = 0;j<n;j++)
            {
                if(i != j)
                {
                    prod *= nums[j];
                }
            }
            res[i] = prod;
        }
        return res;
    }
};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        
        int n = nums.length;
        int i = 1;

        for(int j = 1;j<n;j++)
        { 
            if(nums[i - 1] != nums[j])
            {
                nums[i] = nums[j];
                i++;
            }
        }
        return i;
    }
}